modules/core/api/farm_api.module

```diff
@@ -34,47 +34,8 @@ function farm_api_consumers_list_alter(&$data, $context) {
 function farm_api_entity_base_field_info(EntityTypeInterface $entity_type) {
   $fields = [];
 
-  // Add bundle fields to the consumer entity.
+  // Add allowed_origin field to the consumer entity.
```

Comment:
Minor comment issue: This says `allowed_origin` but should say `allowed_origins`.